tests/test_transform.py

```diff
@@ -177,11 +159,6 @@ def test_search_space_transform_untransform_params() -> None:
         "x8": IntDistribution(2, 4),
         "x9": IntDistribution(1, 10, log=True),
         "x10": IntDistribution(1, 9, step=2),
-        "x11": DiscreteUniformDistribution(0, 1, q=0.2),
-        "x12": UniformDistribution(2, 3),
-        "x13": LogUniformDistribution(1, 10),
-        "x14": UniformDistribution(-2, -2),
-        "x15": LogUniformDistribution(1, 1),
```

Comment:
I just noticed but, don't we have to specify log=True in L157 (`x6`) to make equivalent to x15?

Reply:
You''re right. And same goes for `x5` to be equivalent of `x13`.